report/src/sections/01-introduction.tex

```diff
@@ -1,9 +1,12 @@
 \section{Introduction}\label{sec:introduction}
-\IEEEPARstart{S}{oftware} analysis is a broad field encompassing a wide range of techniques and methodologies, each tailored to address specific aspects of software systems.
-One such technique is value analysis, which focuses on the behavior of a program with respect to its input and output values~\cite{jackson_software_2000}.
-Value analysis can be approached from various angles, including the consideration of meta-information like information flow or taint analysis.
-While these approaches offer valuable insights into the behavior of a program, they often require detailed knowledge of the program's internal workings and are not always applicable to all types of software systems.
-In this paper we use static program analysis, specifically abstract interpretation.
+\IEEEPARstart{S}{oftware} analysis is a broad field encompassing various techniques and methodologies, each tailored to address specific aspects of software systems.
+Static analysis is one such technique, focusing on finding information guaranteed to hold at all executions of a program~\cite{jackson_software_2000}.
+Data flow analysis is a common approach within static analysis, used to examine the data flow through a program.
+In this paper, we employ static program analysis, specifically abstract interpretation, a specialized form of data flow analysis, to analyze software behaviour.
```

Comment:
This is not quite right... abstract interpretation is a framework for creating static analyses.